api/specs/web-server/_computations.py

```diff
@@ -28,6 +28,25 @@ async def get_computation(project_id: ProjectID):
 @router.post(
     "/computations/{project_id}:start",
     response_model=Envelope[_ComputationStarted],
+    responses={
+        status.HTTP_404_NOT_FOUND: {
+            "description": "Project/wallet/pricing details not found"
```

Comment:
Why not adding the model? Perhaps in a separate PR we can start documenting up all these errors properly?

Reply:
Yes, I agree it would be good to add the models. But I think documenting the return statuses is a good first step. That is (for the most part) enough for clients to handle the exceptions which can arise. And digging out these status codes is really difficult TBH. I was thinking this is something one could really use a tool for. In general when doing this exercise of digging out exceptions which can be raised one regrets that annotating exceptions which can be raised by a fcn is not supported in python. That would have been great for these things.

Reply:
I have a script that can dig them out auto-"magically" https://github.com/pcrespov/sandbox-python/blob/fe86d7adf689f9fb0a03ab96d153de39aecee29f/py-exceptions/parse_exceptions.py
Not sure whether it is reliable but it could be a start

Reply:
Right, that is probably a good start